src/test/lumo/lumo/repl_tests.cljs

```diff
@@ -103,7 +104,18 @@
       (is-contains-completion "(MER" "(merge"))
     (testing "JS completions"
       (is-contains-completion "(require 'goog.m" "(require 'goog.math")
-      (is-contains-completion "g/isF" "g/isFunction"))))
+      (is-contains-completion "g/isF" "g/isFunction"))
+    (testing "Spec completions"
+      (testing "namespace fully qualified completion"
+        (s/def ::a-spec string?)
+        (is-contains-completion ":lumo.repl-tests/" ":lumo.repl-tests/a-spec")
+        (reset! s/registry-ref {}))
+      (testing "arbitrary fully qualified keyword"
+        (s/def :arbitrary/a-spec string?)
+        (is-contains-completion ":arbitrary/" ":arbitrary/a-spec")
+        (reset! s/registry-ref {})))))
```

Comment:
does it work for aliased nses?

e.g. `(s/def ::common/foo string?)` and contains a completion for `::common/f`

Reply:
Gonna add a test for that